editor_form.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>DSC Blog - Apply as Editor</title>
+ <link rel="stylesheet" href="./assets/style.css">
+</head>
+<body class="flx">
+
+ <div class="left flx col al-cnt">
+  <form action="">
+    <fieldset>
+      <div class="flx col al-cnt">
+        <legend>APPLY AS EDITOR</legend>
```

Comment:
Change everywhere there's 'Editor' to 'Contributor'

Reply:
Also 
- Reduce the font-size overall on the pages 
- Add more space(padding) on the pages overall across all sides